apps/nova/src/app/[locale]/(dashboard)/problems/new/page.tsx

```
@@ -0,0 +1,105 @@
+      const { data: problem, error: problemError } = await supabase
+        .from('nova_problems')
+        .insert(problemData)
+        .select()
+        .single();
+
+      if (problemError) {
+        throw new Error(problemError.message);
+      }
+
+      // Insert test cases
+      if (problem && problem.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)